Source/UI/NewUI/CommonUI/View/TableViewCell/CKOCellTextField.swift

```diff
@@ -1,65 +1,51 @@
 import UIKit
-import PhoneNumberKit
 
-protocol BillingFormTextFieldCellDelegate: AnyObject {
+protocol CKOCellTextFieldDelegate: AnyObject {
     func updateCountryCode(code: Int)
     func textFieldShouldBeginEditing(textField: UITextField)
     func textFieldShouldReturn()
     func textFieldShouldEndEditing(textField: UITextField, replacementString: String)
     func textFieldShouldChangeCharactersIn(textField: UITextField, replacementString string: String)
 }
 
-final class CKOTextFieldCell: UITableViewCell {
-    weak var delegate: BillingFormTextFieldCellDelegate?
+final class CKOCellTextField: UITableViewCell {
+    weak var delegate: CKOCellTextFieldDelegate?
     var cellStyle: BillingFormCell? = nil
-    var style: CKOTextFieldCellStyle? = nil
+    var style: CKOCellTextFieldStyle? = nil
 
-    
-    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
-        super.init(style: style, reuseIdentifier: reuseIdentifier)
-    }
-    
-    func update(cellStyle:BillingFormCell, style: CKOTextFieldCellStyle, tag: Int) {
-        self.cellStyle = cellStyle
-        self.style = style
-        self.tag = tag
-        setupViewsInOrder()
+    private var mainView: UIView
+    init(mainView: UIView) {
+        self.mainView = mainView
```

Comment:
I am not sure that a cell should be just a wrapper. You're losing the dequeuing behaviour again.

Reply:
I am losing anything. 
https://www.codementor.io/@warrenburton/do-we-need-to-dequeue-tableview-cells-w8szp55uq

Reply:
Updated